Limit stats results

benjaminpkane opened the pull request:
Limits string field histogram results to the first 200, sorted by count in descending order.


```py
import fiftyone as fo
import fiftyone.zoo as foz

F = fo.ViewField

dataset = foz.load_zoo_dataset("cifar100")
dataset.add_sample_field("list_str", fo.ListField, subfield=fo.StringField)

dataset.set_field("list_str", [F("filepath")]).save() # currently crashes the App when `Other fields` is opened

session = fo.Session(dataset)
```

brimoor reviewed (changes requested):
Confirmed that the limiting behavior works as expected, but now the sample tag graphs fail to render when testing with the example snippet you shared.

benjaminpkane commented:
Right, fixed

> Confirmed that the limiting behavior works as expected, but now the sample tag graphs fail to render when testing with the example snippet you shared.

Right, thank you. Fixed.

brimoor reviewed (approved):
LGTM
